docs/abi-spec.rst

```diff
@@ -442,3 +442,22 @@ would result in the JSON:
       "outputs": []
     }
   ]
+
+.. _abi_packed_mode:
+
+Non-standard Packed Mode
+========================
+
+Solidity supports a non-standard packed mode where:
```

Comment:
Perhaps add where it is used and why it is useful - since it is not possible to decode it again when you use more than two dynamically-sized types :-)